elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -327,6 +327,11 @@ public Builder withSubqueryFilterDialect(SubqueryFilterDialect dialect) {
             subqueryFilterDialects.add(dialect);
             return this;
         }
+
+        public Builder withUpdateStatusCode(int updateStatusCode) {
```

Comment:
I don't think we should allow the user to choose an arbitrary status code here and silently "correct" them. Instead, we should be very explicit. `withUpdate200Status` or `withUpdate204Status` or some variety of that flavor.

By default we should retain the existing `204` behavior, of course.